### PR TITLE
[check_config_docstrings.py] improve diagnostics

### PR DESCRIPTION
**Pull request by stas00**
It's difficult to know what to do when one gets an error:

```
python utils/check_config_docstrings.py

Traceback (most recent call last):
  File "utils/check_config_docstrings.py", line 92, in <module>
    check_config_docstrings_have_checkpoints()
  File "utils/check_config_docstrings.py", line 88, in check_config_docstrings_have_checkpoints
    raise ValueError(f"The following configurations don't contain any valid checkpoint:\n{message}")
ValueError: The following configurations don't contain any valid checkpoint:
IdeficsConfig

Exited with code exit status 1
```

After figuring out what it wants, proposing a better assert message.

### CODE REVIEW

File: utils/check_config_docstrings.py [hunk 1]
```diff
@@ -85,7 +85,11 @@ def check_config_docstrings_have_checkpoints():
 
     if len(configs_without_checkpoint) > 0:
         message = "\n".join(sorted(configs_without_checkpoint))
-        raise ValueError(f"The following configurations don't contain any valid checkpoint:\n{message}")
+        raise ValueError(
+            f"The following configurations don't contain any valid checkpoint:\n{message}\n\n"
+            "Hint: ensure to include [foo/bar](https://huggingface.co/foo/bar) link pointing to "
+            "one of the models of this architecture in the docstring of the config classes listed above."
+        )
```
Comment (ydshieh) [inline utils/check_config_docstrings.py:92]: I would prefer to rephrase it like

```
ensure to include a link pointing to one of the models of this architecture in the docstring of the config classes listed above. The link should have a format looks like [foo/bar](https://huggingface.co/foo/bar).
```

**Review by ydshieh — APPROVED** [review]
Thanks!

File: utils/check_config_docstrings.py (hunk 1 above)
Comment (stas00) [reply utils/check_config_docstrings.py:93]: Thank you, @ydshieh 

sure, I used what you suggested and further improved it I hope. Let me know if this is satisfactory.

Comment (ydshieh) [reply utils/check_config_docstrings.py:93]: Way better, thanks a lot. Just a `s` thing in `requirements` maybe. Feel free to merge 🚀

**Pull request merged**